serialization.md

```diff
@@ -26,13 +26,6 @@ This specification uses the following terms:
 	Changing it means creating a new derived image, instead of changing the existing image.
     </dd>
     <dt>
-        Image Filesystem Changeset
-    </dt>
-    <dd>
-        Each layer has an archive of the files which have been added, changed, or deleted relative to its parent layer.
-	Using a layer-based or union filesystem such as AUFS, or by computing the diff from filesystem snapshots, the filesystem changeset can be used to present a series of image layers as if they were one cohesive filesystem.
```

Comment:
On Thu, Sep 01, 2016 at 04:19:26PM -0700, Stephen Day wrote:
> > -	Using a layer-based or union filesystem such as AUFS, or by computing the diff from filesystem snapshots, the filesystem changeset can be used to present a series of image layers as if they were one cohesive filesystem.
> 
> Let's merge this information into the description of layer.

It's already covered here [1] and is not important for defining the
term, so I'd rather leave it out.

[1]: https://github.com/opencontainers/image-spec/blame/08ef0b1d662ac889d3bf2e99957acddad8e5804f/layer.md#L32